app/models/item.rb

```diff
@@ -0,0 +1,70 @@
+class Item < ApplicationRecord
+  belongs_to :brand
+  belongs_to :item_family
+  belongs_to :variety
+
+  validates :added_sugar, numericality: { greater_than_or_equal_to: 0 }, presence: true
```

Comment:
None of these nutrient values should be required. This would make it either extremely annoying to manually add an item or we'd end up with really bad data.